### DISCUSSION TITLE
Status of QNX support

### DISCUSSION THREAD

**Discussion by ccawley2011** [open-watcom/open-watcom-v2#933] [General]
It seems that OpenWatcom still has support for QNX 4, however the relevant headers and libraries aren't currently provided in CI builds. What state is this in currently, and is there any way to enable QNX support in local builds?

**Comment by jmalak** [comment]
Yes, QNX support is still in source tree and build system support to build QNX version.
But it is completely untested, time-to-time a try to compile it (I have a copy of old QNX header files) but have no libraries to create executable. If you want to touch with QNX then you can use following procedure to compile QNX. You can use only Watcom compilers for building QNX stuff no other compilers as gcc etc..
- first you need copy of QNX header files to **`bld/hdr/qnx`** , but take into account that some of them are Watcom and will be overwriten by OW build
- you will need to add QNX libraries to WATCOM bootstrap compiler to be able create executable, probably you will need to modify **`bld/build/bwlink.lnk`** script for your QNX library location
-  generaly if you create bootstrap tools then QNX stuff can be created by command **` builder build os_qnx `** in **` bld `** folder
